lib/Router.php

```diff
@@ -266,19 +266,19 @@ public function route(string $method, string $uri, ...$actions): Router {
         }
 
         $actions = array_merge($this->actions, $actions);
+        $methods = is_array($methods) ? $methods : [$methods];
```

Comment:
The current approach is actually the cleaner one. `(array) $methods` will also happily cast objects to an array instead of putting the object into that array.